docs/guidelines/dockerfile-linting.md

```diff
@@ -0,0 +1,79 @@
+# Dockerfile linting
+
+From [hadolint](https://github.com/hadolint/hadolint):
+> A smarter Dockerfile linter that helps you build [best practice](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices) Docker images. The linter parses the Dockerfile into an AST and performs rules on top of the AST. It stands on the shoulders of [ShellCheck](https://github.com/koalaman/shellcheck) to lint the Bash code inside RUN instructions.
+
+## CI/CD
+
+A great way to ensure that your any changes being brought into your repository complies with the best practices is to add the linting on pull requests as a part of the code review
+
+Below is an example workflow that will run linting on our dockerfiles whenever there is a pull request that contains changes to any dockerfiles.
+
+```yml
+name: Running linting on dockerfile
```

Comment:
This is the name of the action that will be visible in the "Actions" pane of the repository. It is limited in horizontal space, so it is a good idea to use short and descriptive names. Instead of using continuous tense, I recommend describing the purpose of the step. 

For example, use a name like "**Lint Dockerfile**".

Reply:
Changed to your recommendation